packages/docs/src/styles/_navbar.scss

```diff
@@ -0,0 +1,66 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+
+@import "variables";
+@import "../../../core/src/components/button/common";
+
+/*
+All the components that live *inside* the navbar.
+
+.docs-navbar styles live in _layout.scss
```

Comment:
i left layout-specific styles in _layout.scss, which is arguably the entire `.docs-navbar` selector